seeding/user.seeding.js

```diff
@@ -6,15 +6,17 @@ import AddressFactory from '../test/factories/address.factory';
 import User from '../server/models/user.model';
 import { USER_ROLE } from '../server/helpers/constants';
 import { logLoading, logError, logTable } from './seed-helpers';
-import { ROLE } from './seed-constants';
-import { hashPassword } from '../server/services/user.service';
+import { ROLE, DEFAULT_PASSWORD } from './seed-constants';
+import { hashPassword, generateReferralCode } from '../server/services/user.service';
 import { USER_DEFAULTS } from './defaults.seeding';
 
 export const seedUsers = async (limit, role, customValues = {}) => {
   const roleValue = Object.keys(ROLE).find((key) => ROLE[key] === role);
 
-  const password = customValues.password || USER_DEFAULTS.PASSWORD;
+  const password = customValues.password || USER_DEFAULTS?.password || DEFAULT_PASSWORD;
```

Comment:
If you implement the `userDefaults` suggestion above, then this can be rewritten as:

```suggestion
const password = userDefaults?.password || DEFAULT_PASSWORD;
```